src/redux/reducers/storeLocation.reducer.js

```diff
@@ -0,0 +1,12 @@
+import { STORE_LOCATION_ACTIONS } from "../actions/storeLocation.actions"
+
```

Comment:
Hello... I've added default state but left it as an empty array for now, as I'm not sure how much detail we want to add to the store.